Python/Product/Debugger.Concord/Proxies/StructProxy.cs

```diff
@@ -20,6 +20,7 @@
 using Microsoft.Dia;
 using Microsoft.PythonTools.Common.Parsing;
 using Microsoft.VisualStudio.Debugger;
+using Microsoft.VisualStudio.Debugger.Interop;
```

Comment:
Is this `using` necessary? Seems weird that code was deleted, but a using was added?

Reply:
Oh that's probably a mistake. I can just remove that.